### ISSUE TITLE
Changing the supertype in the CND is not enough

### ISSUE THREAD

**Issue by matsodolphijkvk** [bloomreach/docs-feedback#116]
Goodmorning,

I have a suggestion for article: https://xmdocumentation.bloomreach.com/library/concepts/content-repository/configure-document-versioning.html

We added the [audittrace ](hippostdpubwf:audittrace)  supertype in the CND. After the change we ran into bootstrapping problems in our dev/test/acc/prod environments. Turns out that just adding the supertype in the CND is not enough. The changes also needed to be made in the namespace yaml's of the documents. This made perfect sense in hindsight but it can't hurt mentioning this in the documentation as well. And it might prevent others from running into the same issues as us.

Kind regards,

Mats

**Comment by nvankampenhout** [comment]
Thanks for the feedback @matsodolphijkvk ! I'll update the documentation asap.

**Comment by nvankampenhout** [comment]
I wasn't able to reproduce the issue with a plain vanilla archetype project but I can imagine it depends on the specific bootstrap data of a project.

I added a note at the bottom of https://xmdocumentation.bloomreach.com/library/concepts/content-repository/configure-document-versioning.html.

**Comment by matsodolphijkvk** [comment]
Goodmorning @nvankampenhout, I now think that the error was caused by adding the supertype in the CND. After a deploy on an existing environment we got this bootstrap error: 

25.01.2023 15:51:16 [main] ERROR [org.onehippo.cm.engine.ConfigurationServiceImpl.applyConfig():978] Failed to apply config
javax.jcr.RepositoryException: The following node type change contains non-trivial changes.Up until now only trivial changes are supported. (see javadoc for org.apache.jackrabbit.spi.commons.nodetype.NodeTypeDefDiff):
org.hippoecm.repository.jackrabbit.HippoNodeTypeRegistry$CustomNodeTypeDefDiff[
	nodeTypeName={http://www.kvkwebsite.com/kvkwebsite/nt/1.0}basedocument,
	mixinFlagDiff=NONE,
	supertypesDiff=MAJOR,
	propertyDifferences=[
	],
	childNodeDifferences=[
	]
]
I've omitted the rest of the stacktrace. The bootstrapping process halted after this error and new content and config was missing from the environment. Perhaps this makes more sense than the missing supertype in the bootstrap yaml.

Kind regards,

Mats
